Google Play Music/UpdateDialog.cs

```diff
@@ -80,20 +80,19 @@ public UpdateDialog(string changeLog, string oldVersion, string newVersion)
 
         private void InitializeComponent()
         {
-            // Auto generated by the designer
-            notNowButton = new MaterialRaisedButton();
-            this.updateButton = new MaterialRaisedButton();
-            this.changeLogText = new MaterialLabel();
+            this.notNowButton = new MaterialSkin.Controls.MaterialRaisedButton();
+            this.updateButton = new MaterialSkin.Controls.MaterialRaisedButton();
+            this.changeLogText = new MaterialSkin.Controls.MaterialLabel();
```

Comment:
It looks here as if your Visual Studio is configured differently to mine, all these namespace paths really aren't required as they are all in the "using".  Check your options so that this doesn't happen.